### ISSUE TITLE
2-02 programs additions and updates

### ISSUE THREAD

**Issue by anthoninlize** [wmo-im/wmds#325]
Hello

# **Branch**

\[add branch when created\]

# **Summary and Purpose**
New additions and udpates to the 2-02 code list for Programs from OceanOPS. Required for synchronisation of marine observing elements.

# **Stakeholder(s)**

support@ocean-ops.org : The OceanOPS team

# **Proposal**

Please see the proposal in the Excel file attached. Different tabs, one per missing programs inside the observing networks and a tab to update existing program paths and definitions.
They all contain a notation, path, name and description.

# **Reason**

The additions are required to submit observing facilities within those programs, otherwise they are rejected.
The updates are required to better reflect the contributors (wrong hierarchy and definitions.

Thank you

[20210701-WIGOSPrograms.xlsx](https://github.com/wmo-im/wmds/files/6749498/20210701-WIGOSPrograms.xlsx)

**Comment by fstuerzl** [comment]
Branch created: https://github.com/wmo-im/wmds/blob/issue325/tables_en/2-02.csv, view diffs https://github.com/wmo-im/wmds/compare/issue325

|notation|path|name|description|
|--|--|--|--|
|jamstecMB|\Co-sponsored\GOOS\DBCP\JAMSTEC-MB|JAMSTEC-MB|Japan Agency for Marine-Earth Science and Technology Moored Buoy|
|BOMMB|\Co-sponsored\GOOS\DBCP\BOM MB|BOM MB|BOM Moored Buoy|
|AARI|\Co-sponsored\GOOS\DBCP\AARI|AARI|Buoys managed by Arctic and Antarctic Research Institute, Russian Federation|
|SASDB|\Co-sponsored\GOOS\DBCP\SAS-DB|SAS-DB|Voluntary buoys deployed by Sailing and Science|
|DFOUSVs|\Co-sponsored\GOOS\DBCP\DFO USVs|DFO USVs|Unmanned/Autonomous Surface Vehicles by DFO, Canada|
|NZTSU|\Co-sponsored\GOOS\DBCP\NZ-TSU|NZ-TSU|GNS Science is a New Zealand Crown Research Institute|
|peruMB|\Co-sponsored\GOOS\DBCP\Peru-MB|Peru-MB|Moored Buoys deployed by SENAMHI-Peru|
|USACE|\Co-sponsored\GOOS\DBCP\USACE|USACE|Owned and maintained by U.S. Army Corps of Engineers, Data provided by Scripps Institution of Oceanography, Waverider Buoy|
|SBUMB|\Co-sponsored\GOOS\DBCP\SBU-MB|SBU-MB|Owned and maintained by Stony Brook University Buoy|
|UNHMB|\Co-sponsored\GOOS\DBCP\UNH-MB|UNH-MB|Owned and maintained by the University of New Hampshire, Moored Buoy|
|OOIMB|\Co-sponsored\GOOS\DBCP\OOI-MB|OOI-MB|Station owned and maintained by Ocean Observatories Initiative, Buoy|
|CSIMB|\Co-sponsored\GOOS\DBCP\CSI-MB|CSI-MB|Owned and maintained by UNC Coastal Studies Institute, Data provided by Scripps Institution of Oceanography|
|NERACOOSMB|\Co-sponsored\GOOS\DBCP\NERACOOS-MB|NERACOOS-MB|Owned and maintained by Woods Hole Group/NERACOOS, Data provided by Scripps Institution of Oceanography|
|COMPSMB|\Co-sponsored\GOOS\DBCP\COMPS-MB|COMPS-MB|Owned and maintained by COMPS (University of South Florida) Offshore Buoy|
|TAMUMB|\Co-sponsored\GOOS\DBCP\TAMU-MB|TAMU-MB|Developed and maintained by Texas A&M University, Funding provided by the Texas General Land Office's Oil Spill Prevention & Response Program|
|ILBD|\Co-sponsored\GOOS\DBCP\IL-DB|IL-DB|Drifting buoy by Israel|
|ILMB|\Co-sponsored\GOOS\DBCP\IL-MB|IL-MB|Moored buoys by Israel|

**Comment by echarpent** [comment]
No comment or concern from my perspective; suggest getting the greenlight from GOOS/OCG via Champika Gallage (cgallage@wmo.int)

**Comment by champikagallage** [comment]
I agree with the new content provided by OceanOPS on new programme list. It is great if this can go through the fast track process without delay. Thanks

**Comment by joergklausen** [comment]
reviewed and approved

**Comment by amilan17** [comment]
@anthoninlize should the notation of "jamstecMB" be all caps?

**Comment by anthoninlize** [comment]
@amilan17 Indeed, referring to the [documentation](https://github.com/wmo-im/wmds/wiki/Guidance-for-spelling-of-notation). I updated the proposal above.

**Comment by amilan17** [comment]
Thanks. I think I'll have a lot more questions. For example is it "Sailing and Science" or "Sailing for Science"? reference: https://public.wmo.int/en/media/news/sailing-science

**Comment by amilan17** [comment]
More questions: 

1.  BOMMB: Bureau of Meteorology? 
2.  I don't think we should include descriptions about the missions of these programs. A simple breakdown of the acronym or _brief_ explanation is sufficent. e.g. (@joergklausen -- please confirm)
3.  don't include URLs in the descriptions

**Comment by joergklausen** [comment]
> 2. I don't think we should include descriptions about the missions of these programs. A simple breakdown of the acronym or _brief_ explanation is sufficent. e.g. (@joergklausen -- please confirm)
> 3. don't include URLs in the descriptions

I am neutral on both points. Some description is useful to _identify_ the program. Several examples are a bit too verbose for my taste, but I wouldn't be rigorous here, because these descriptions are not critical for _defining_ a term. As to URLs, if they are stable, they might be useful to include, but they are certainly not mandatory.

**Comment by joergklausen** [comment]
@anthoninlize Can the descriptions be shortened to 2-4 lines max (lines as displayed in the table at the top)? That would be more in line with all the rest, where the description is essentially a long name of the program mentioned.

**Comment by anthoninlize** [comment]
@joergklausen @amilan17 I've updated as far as I could, since I'll never have the feedback in time from my team (I just received feedback for things I asked two weeks ago).

**Comment by amilan17** [comment]
@anthoninlize - thanks.

**Comment by amilan17** [comment]
@anthoninlize what did you update? Just the proposal summary in this issue?

**Comment by anthoninlize** [comment]
@amilan17 yes, the table in the proposal.

**Comment by amilan17** [comment]
@anthoninlize There are entries for OGSGlider. We cannot repeat notations. Can you please respond to this comment with the correction? 

<table><tbody><tr><td>OGSGlider</td><td>\Co-sponsored\GOOS\OceanGliders\Adriatic Sea Observatory</td><td>Adriatic Sea Observatory</td><td>Repeated section in the Adriatic water transformation region</td></tr><tr><td>OGSGlider</td><td>\Co-sponsored\GOOS\OceanGliders\OGS glider program</td><td>OGS glider program</td><td>Istituto Nazionale di Oceanografia e di Geofisica Sperimentale glider program.</td></tr></tbody></table>

**Comment by anthoninlize** [comment]
> @anthoninlize There are two entries for OGSGlider. We cannot repeat notations. Can you please respond to this comment with the correction? 
> 
> OGSGlider	\Co-sponsored\GOOS\OceanGliders\Adriatic Sea Observatory	Adriatic Sea Observatory	Repeated section in the Adriatic water transformation region
> OGSGlider	\Co-sponsored\GOOS\OceanGliders\OGS glider program	OGS glider program	Istituto Nazionale di Oceanografia e di Geofisica Sperimentale glider program.

That's a mistake. Here's an updated version:
<table><tbody><tr><td>ASOGlider</td><td>\Co-sponsored\GOOS\OceanGliders\Adriatic Sea Observatory</td><td>Adriatic Sea Observatory</td><td>Repeated section in the Adriatic water transformation region</td></tr><tr><td>OGSGlider</td><td>\Co-sponsored\GOOS\OceanGliders\OGS glider program</td><td>OGS glider program</td><td>Istituto Nazionale di Oceanografia e di Geofisica Sperimentale glider program.</td></tr></tbody></table>

**Comment by joergklausen** [comment]
I think we should avoid dashes '-' in a notation, such as in 'OOI-EnduranceGlider'. Can this (and other occurences) be renamed to 'OOIEnduranceGlider', please?

**Comment by anthoninlize** [comment]
Agreed, I don't know why I let this happened... I just updated the proposal above removing all the dashes '-' and underscores '_' in the notations.

**Comment by amilan17** [comment]
@anthoninlize @joergklausen please take a moment to review the diff of changes

https://github.com/wmo-im/wmds/pull/376/files
